circleci: failed tests: TestNodeRestart

The following test appears to have failed:

[#22694](https://circleci.com/gh/cockroachdb/cockroach/22694):

```
I160913 20:19:51.052711 99 kv/dist_sender.go:1125  [n2] range 1: caching new lease holder {NodeID:1 StoreID:1 ReplicaID:1}
W160913 20:19:51.067897 96 kv/dist_sender.go:695  [n2] 1 retries for an RPC at sequence 4, last error was: <nil>, remaining key ranges {"/Table/2/1/1/\"eventlog\"/3/1" "/Table/2/1/1/\"eventlog\"/3/1/NULL"}: [txn: 1d1b5ef7], Get [/Table/2/1/1/"eventlog"/3/1,/Min)
I160913 20:19:51.069510 109 rpc/clock_offset.go:151  [n2] 3 of 3 nodes are within the maximum offset of 250ms
I160913 20:19:51.075062 96 kv/dist_sender.go:1051  [n2] application error: range 3: replica {2 2 2} not lease holder; node_id:1 store_id:1 replica_id:1  is
I160913 20:19:51.075130 96 kv/dist_sender.go:1125  [n2] range 3: caching new lease holder {NodeID:1 StoreID:1 ReplicaID:1}
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
	panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x88f330]

goroutine 42 [running]:
panic(0x150a7c0, 0xc420010060)
	/usr/local/go/src/runtime/panic.go:500 +0x1a1
github.com/cockroachdb/cockroach/util/stop.(*Stopper).Recover(0xc4203e9560)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:176 +0x6e
panic(0x150a7c0, 0xc420010060)
	/usr/local/go/src/runtime/panic.go:458 +0x243
github.com/cockroachdb/cockroach/storage.(*Store).processRaftRequest(0xc420198280, 0x7f8784f24888, 0xc4203c7890, 0xc420300280, 0x0)
	/go/src/github.com/cockroachdb/cockroach/storage/store.go:2290 +0x170
github.com/cockroachdb/cockroach/storage.(*Store).processRequestQueue(0xc420198280, 0x7)
	/go/src/github.com/cockroachdb/cockroach/storage/store.go:2551 +0x137
github.com/cockroachdb/cockroach/storage.(*raftScheduler).worker(0xc4201467e0, 0xc4203e9560)
	/go/src/github.com/cockroachdb/cockroach/storage/scheduler.go:194 +0x331
github.com/cockroachdb/cockroach/storage.(*raftScheduler).Start.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/scheduler.go:160 +0x33
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4203e9560, 0xc4202ccf00)
I160913 20:19:55.200374 439 acceptance/chaos_test.go:416  1113 219.0/sec
I160913 20:19:55.301636 439 acceptance/cluster/localcluster.go:679  stopping
I160913 20:19:55.301858 460 acceptance/cluster/localcluster.go:579  event stream done, resetting...: net/http: request canceled
I160913 20:19:55.301876 460 acceptance/cluster/localcluster.go:602  events monitor exits
I160913 20:19:56.220726 439 acceptance/cluster/localcluster.go:722  node 1: stderr at /tmp/circle-artifacts.yd0u06k/acceptance/TestNodeRestart/roach1/stderr.2016-09-13T20_19_56Z.log
--- FAIL: TestNodeRestart (10.07s)
	localcluster.go:658: unexpected extra event &{1 die} (after [{1 die} {1 restart}])
=== RUN   TestContinuousLoad_BlockWriter
--- SKIP: TestContinuousLoad_BlockWriter (0.00s)
	util_test.go:366: skipping since not run against remote cluster
=== RUN   TestContinuousLoad_Photos
--- SKIP: TestContinuousLoad_Photos (0.00s)
	util_test.go:366: skipping since not run against remote cluster
=== RUN   TestEventLog
--- SKIP: TestEventLog (0.00s)
	event_log_test.go:36: flaky: #8562
--
--- SKIP: TestBuildBabyCluster (0.00s)
	terraform_test.go:30: only enabled during testing
=== RUN   TestFiveNodesAndWriters
--- SKIP: TestFiveNodesAndWriters (0.00s)
	util_test.go:366: skipping since not run against remote cluster
FAIL
ok github.com/cockroachdb/cockroach/acceptance 1337s

```
Please assign, take a look and update the issue accordingly.

--- a/storage/store.go
+++ b/storage/store.go
@@ -2287,7 +2287,7 @@ func (s *Store) processRaftRequest(
 			panic(fmt.Sprintf("unexpected quiesce: %+v", req))
 		}
 		status := r.RaftStatus()
-		if status.Term == req.Message.Term && status.Commit == req.Message.Commit {
+		if status != nil && status.Term == req.Message.Term && status.Commit == req.Message.Commit {
 			r.setQuiescent(true)
 			return
 		}

